AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
 Setting up an A -> B Replication Configuration with Empty Databases
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-On A:
+**On A**:
+
+* Source the ydbenv environment file.
+
+.. parsed-literal::
+   source ./ydbenv A r122
+
+* Create the database and set up replication.
+
+.. parsed-literal::
+   $./db_create
+   %GDE-I-LOADGD, Loading Global Directory file
+        /home/A/yottadb.gld
+   %GDE-I-VERIFY, Verification OK
+
+   %GDE-I-EXECOM, Executing command file /home/A/gdemsr
```

Comment:
/home/A indicates that A is a user id. Might be confusing. Better to say /home/user/A instead. Same with other /home/ usages.

Reply:
Changed